modules/apps/object/object-api/src/main/java/com/liferay/object/constants/ObjectFieldSettingConstants.java

```diff
@@ -23,4 +23,7 @@ public class ObjectFieldSettingConstants {
 
 	public static final String NAME_STATE_FLOW = "stateFlow";
 
+	public static final String OBJECT_DEFINITION1_SHORT_NAME =
```

Comment:
I'm not sure if the 1 should be together, like "_DEFINITION1_". Could you please search usages?